notebooks/concurrent_tasks.ipynb

```diff
@@ -0,0 +1,1224 @@
+{
```

Comment:
<p>Short explanation above this cell to help users know we are first writing documents into a database?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
done